packages/genesys-spark-components/src/components/stable/gux-icon/gux-icon.types.ts

```diff
@@ -592,6 +595,12 @@ export type GuxIconIconName =
   | 'refresh'
   | 'reset'
   | 'resource'
+  | 'restricted/fa/building-user-regular'
+  | 'restricted/fa/calendar-circle-user-regular'
+  | 'restricted/fa/compass-drafting-regular'
+  | 'restricted/fa/grid-2-regular'
+  | 'restricted/fa/headset-regular'
+  | 'restricted/fa/screwdriver-wrench-regular'
```

Comment:
My thinking on this was that it seems important to flag the restriction in the icon name but also open to other views here.